backend/src/zelthy/apps/shared/platformauth/models.py

```diff
@@ -106,8 +106,14 @@ def create_user(
                     )
                 else:
                     if not cls.validate_password(password):
-                        message = "Invalid password. Password must follow rules xyz"
-                    else:
```

Comment:
@AjaySDwivedi1 We don't need to remove the `else` block here, we can just update the `message` string